airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -1273,4 +1254,47 @@ protected AirbyteMessage createStateMessage(final DbStreamState dbStreamState, f
     }
   }
 
+  /**
+   * Method to add state_type: "standard" to state message for CTID based syncs
+   *
+   * @param syncState
+   */
+  protected void addStandardStateTypeToSyncState(final JsonNode syncState) {
+    // no op
+  }
+
+  /**
+   * Read catalog to build iterators for the first sync
+   *
+   * @param configuredCatalog catalog of DB source
+   * @return A list of Airbyte messages generated during sync
+   * @throws Exception
+   */
+  protected List<AirbyteMessage> readCatalogForInitialSync(final ConfiguredAirbyteCatalog configuredCatalog) throws Exception {
+    return MoreIterators
+        .toList(source.read(config, configuredCatalog, createEmptyState(streamName, getDefaultNamespace())));
```

Comment:
Does this need to be overwritten? What happens if the base method just uses null?